### PR TITLE
Add GitHub Actions Workflow to remove labels from closed issues

### LINKED ISSUE #7967: Add GH Action to remove status labels when closed
### First check

- [X] I am a contributor to the Prefect codebase

### Description

When issues are closed, the status label should be removed. No reason to leave "status:triage" on a closed issue.

### Impact

This will simplify counts of issues in various states and avoid misleading statuses on archived issues.

### Additional context

This will also put us in a good place to make other changes to issue labels with automations, e.g. we can remove `status:triage` when `status:accepted` is applied or add `status:in-progress` when someone is assigned / a pull request is opened.

### PR DESCRIPTION
**Pull request by mohitsaxenaknoldus**
Closed #7967 

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: .github/workflows/remove-issue-labels.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    types: [closed]
+
+jobs:
+  remove_label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-remove-labels@v1
+        with:
+          labels: status:triage
```
Comment (madkinsz) [inline .github/workflows/remove-issue-labels.yaml:14]: I'm hesitant to add a dependency on an external action — do you think you could use the `gh` CLI instead? I think that'd make it easier for us to extend this in the future too.

**Review by madkinsz — COMMENTED** [review]
Sweet thanks for contributing this!

**Comment by mohitsaxenaknoldus** [comment]
@madkinsz Check now, please.

**Review by madkinsz — APPROVED** [review]
Thanks!

**Pull request merged**